app/epa/settings.py

```diff
@@ -37,16 +48,15 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = os.getenv(
-    "EPA_SECRET_KEY", "v@p9^=@lc3#1u_xtx*^xhrv0l3li1(+8ik^k@g-_bzmexb0$7n"
+SECRET_KEY = env(
+    "EPA_SECRET_KEY", default="v@p9^=@lc3#1u_xtx*^xhrv0l3li1(+8ik^k@g-_bzmexb0$7n"
 )
 
 ALLOWED_HOSTS = ["*"]
 
-CSRF_TRUSTED_ORIGINS = [
-    f"https://{os.getenv('TRUSTED_HOST')}",
-    f"http://{os.getenv('TRUSTED_HOST')}",
-]
+trusted_host = env("TRUSTED_HOST", default=None)
+if trusted_host is not None:
+    CSRF_TRUSTED_ORIGINS = [f"https://{trusted_host}", f"http://{trusted_host}"]

```

Comment:
Suggestion: `CSRF_TRUSTED_ORIGINS = env.list('CSRF_TRUSTED_ORIGINS')` -> looks like this in env: `localhost,127.0.0.1` or similar. This avoids trusting unsecure connection.

Reply:
What I like with `"TRUSTED_HOST"` is that there is only one trusted host, whereas with your suggestion I could provide a list

Reply:
CSRF_TRUSTED_ORIGINS is _always_ a list. Previously, only one host was specified (or only one host could be specified), but internally, this was secretly converted into two trusted origins: one HTTPS and one unencrypted HTTP. With my suggestion, this hidden conversion (and, in my opinion, security vulnerability) would disappear. Instead, all trusted origins can and must now be specified explicitly.

Reply:
You got a point here. I suggest you implement your suggestions in this PR (@paulapreuss was done with her part on it)